SinaiStCatherine/SinaiEt006.xml

```diff
@@ -0,0 +1,707 @@
+                   ፈሪሀ፡ እግዚአብሔር። ወምክር፡ ሠናይት፡ ለኩሉ፡ ዘይገብራ፡ ወስብሐቲሁኒ፡ ይነብር፡ ለዓለመ፡ ዓለም፡ አሜን።
+                    ሐሳብ፡ ቀዳሚ፡ ዘነገሩ፡ አበዊነ፡ ዘከመ፡ ለሐኮሙ፡ ለአቡነ፡ አዳም፡ ወለሔዋን፡</incipit>
+                     <note>Probably incomplete, the text on <locus target="#199vb"/>
+                   is erased or cannot be read in the microfilm.</note>
+           </msItem>
+
+                   <msItem xml:id="ms_i4">
+                       <locus target="#200r"/>
+                       <title type="complete" ref="LIT6011FeastsList">A version of the list of church feasts,
+                       with the names of the months in Arabic transcribed in Ethiopic, starting with
+                     <foreign xml:lang="gez">ኤሎል</foreign></title>
+                       <textLang mainLang="gez"/>
+                    </msItem>
+
+                <msItem xml:id="ms_i5">
+                  <locus from="200r" to="203v"/>
+               <title type="complete" ref="LIT6291HassabaRetu">A version of the treatise on computus and calendar that sometimes bear
+               the title <foreign xml:lang="gez">Ḥassāba rәtuʿ</foreign>.
+                It includes such subparts as Explanation of the Calculation of Henoch,
+             (it seems to be akin to the text II.17 from the Comboni fragment
+             (Nosnitsin 2021:55-56), <foreign xml:lang="gez">ሐሳበ፡ ፀሐይ፡</foreign>,
+            <foreign xml:lang="gez">ሐሳበ፡ በረከት፡</foreign>,
+         <foreign xml:lang="gez">ሐሳበ፡ መዋዕል፡ ወትንቢት፡ ዘኤልያስ፡</foreign>.</title>
+        <textLang mainLang="gez"/>
+          </msItem>
+
+
+
+
+     <msItem xml:id="ms_i6">
+    <locus from="207r" to="208r"/>
+    <title type="complete" ref="LIT4858Litany">Litany (liṭon) for Tuesday</title>
+    <textLang mainLang="gez"/>
+    <incipit xml:lang="gez"><gap reason="lost" extent="6" unit="characters"/>
+    ስምከ፡ ሕያው፡ ዘኢይመውት፡ ትጉህ፡ <gap reason="lost" extent="3" unit="characters"/> ዘኢይደክም፡
+      እግዚእ፡ ስሉጥ፡ ለዕለ፡ ኵሉ፡ አምላክ፡ ዘኢይትኀባእ፡ ዘኪያከ፡ ያንቀያዓዱ፡ ኵሉ፡ ፍጥረት። ዘበሰማይ፡ ወዘበ፡ ምድር፡ ዘፈጠርከ፡
+  </incipit>
+   </msItem>
+
+   <msItem xml:id="ms_i7">
+  <locus from="208r" to="211v"/>
+  <locus from="3r" to="4v"/>
+  <title type="incomplete" ref="LIT1990Mawase">Excerpts from the Mawāśǝʾǝt
+   with no musical notation, written in the main <ref target="#h1">Hand 1</ref>.
+   It appears that the scribe first copied
+ the text on the last leaves of the textblock and continued writing
+ in the front guard quire.</title>
+  <textLang mainLang="gez"/> 
+      <incipit xml:lang="gez"><hi rend="rubric">፲መዋሥእት፡ ዘጻድቃን፡ ዘ፳፩፡</hi>
+      ዘኪሩቤል፡ ሰማይ፡ ይጸርኁ፡ ወይሰጠዉ፡ ቃለ፡ በኩራ፡ ለድንግል፡ ለጻድቃን፡ ክብር።
+      <gap reason="ellipsis" resp="DN"/><pb n="3r"/>
+      <gap reason="lost" extent="20" unit="charcters"/> ሕፃን፡ ንኡስ፡ ቂርቆስ፡
+      ቅዱስ፡ ነቅዕ፡ ዘእምዐዘቅት፡ ማይ፡ ዘኢይማስን፡
+    </incipit>
+ </msItem>
+
+ <msItem xml:id="ms_i8">
+ <locus from="211v" to="212r"/>
+ <title type="incomplete">A portion of the pascal tables (one table only),
```

Comment:
Computational tables for calculating the day of Easter? (never heard of pascal tables, unless they are coded in the Pascal language...)

Reply:
ok